### ISSUE TITLE
Newlines '\n' Isn't Getting Stripped From The Passwords Returning Bad Results With Valid Stuff.

### ISSUE THREAD

**Issue by DEMON1A** [DEMON1A/GmailBruterV2#3]
- I'm currently fixing this tool instead of deleting it. I wanted to share to hall process with you guys using the issues.

- On the `StartBruteAccount()` Function at line *24* the program is taking the passwords from a newline-separated list without removing the newline from it. that uses a bad password won't be valid with the authentication and even if the password is valid the program won't show that it's valid since it's using a bad password. the fix will be something like this

```
for Password In PasswordsList:
    Password = Password.rstrip("\n")
    
    [SMTP Auth Here...]
```

**Comment by DEMON1A** [comment]
4117bca

**Comment by DEMON1A** [comment]
- Closing this issue since it has been solved on in 40e156d please update the tools.
